### PR TITLE
PLAT-48382: Upgrade Gatsby to v1

### PR DESCRIPTION
**Pull request by webOS101**
Migrate to Gatsby v1

### CODE REVIEW

File: src/layouts/docs.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import '../css/github.css';
+
+export default class SiteTemplate extends React.Component {
+	static propTypes = {
+		children: PropTypes.func,
+		data: PropTypes.object,
+		location: PropTypes.object
+	}
+
+	// <SiteSection accent="1">
```
Comment (sugardave) [inline src/layouts/docs.js]: Safe to delete this example?

File: src/layouts/markdown.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import '../css/github.css';
+
+export default class SiteTemplate extends React.Component {
+	static propTypes = {
+		children: PropTypes.func,
+		data: PropTypes.object,
+		location: PropTypes.object
+	}
+
+	// <SiteSection accent="1">
```
Comment (sugardave) [inline src/layouts/markdown.js]: Safe to delete this example?

File: src/layouts/docs.js (hunk 1 above)
Comment (webOS101) [reply src/layouts/docs.js]: Sure.

File: src/layouts/markdown.js (hunk 2 above)
Comment (webOS101) [reply src/layouts/markdown.js]: Sure.

**Pull request merged**